playwright.config.ts

```diff
@@ -12,7 +12,7 @@ import { defineConfig, devices } from '@playwright/test';
  * See https://playwright.dev/docs/test-configuration.
  */
 export default defineConfig({
-  testDir: './app/tests',
+  testDir: './e2e',
```

Comment:
bun doesn't support excluding some folders atm so I had to move e2e tests out of the app and configure bun to only test app folder (via bunfig.toml file)